internal/expr/prepare.go

```diff
@@ -291,3 +418,32 @@ func (pe *ParsedExpr) Prepare(args ...any) (expr *PreparedExpr, err error) {
 
 	return &PreparedExpr{inputs: inputs, outputs: outputs, sql: sql.String()}, nil
 }
+
+// printCols prints a bracketed, comma seperated list of fullNames.
+func printCols(cs []fullName) string {
+	var s bytes.Buffer
+	s.WriteString("(")
+	for i, c := range cs {
+		s.WriteString(c.String())
+		if i < len(cs)-1 {
+			s.WriteString(", ")
+		}
+	}
+	s.WriteString(")")
+	return s.String()
+}
+
+// namedParams returns n incrementing parameters begining at start.
+func namedParams(start int, n int) string {
```

Comment:
This function feels very similar to `printCols`, should the name be consistent with the above?

Reply:
I've changed the name to `generateNamedParams` and added an example in the comment to make it clear what it actually does.